[2022-10-28 05:44:30,477: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 05:44:30,479: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 05:44:30,479: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 05:44:30,479: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 05:44:32,780: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 05:47:09,363: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 05:47:09,365: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 05:47:09,365: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 05:47:09,365: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 05:47:11,225: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 06:13:37,684: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 06:13:37,686: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 06:13:37,686: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 06:13:37,686: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 06:13:39,569: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 08:01:20,304: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:01:20,304: ERROR: stage_02_featurization]: name 'read_yaml' is not defined
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 50, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 22, in main
    config = read_yaml(config_path)
NameError: name 'read_yaml' is not defined
[2022-10-28 08:02:03,620: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:02:03,622: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:02:03,622: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:02:03,622: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:02:03,622: ERROR: stage_02_featurization]: 'max_features'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 50, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 37, in main
    max_features = params['featurize']['max_features']
KeyError: 'max_features'
[2022-10-28 08:02:46,228: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:02:46,229: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:02:46,229: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:02:46,229: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:02:47,005: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 08:07:31,748: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:07:31,749: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:07:31,751: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:07:31,751: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:07:31,929: INFO: common]: The input data frame artifacts\prepared\train.tsv size is (19921, 3)
[2022-10-28 08:07:31,929: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 08:08:27,006: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:08:27,007: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:08:27,007: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:08:27,007: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:08:27,181: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 08:08:27,193: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 08:10:04,135: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:10:04,136: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:10:04,136: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:10:04,136: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:10:04,306: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 08:10:04,322: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 08:16:53,739: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:16:53,740: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:16:53,741: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:16:53,741: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:16:53,908: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 08:16:53,923: ERROR: stage_02_featurization]: name 'np' is not defined
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 55, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 42, in main
    train_words = np.array(df_train.text.str.lower().values.astype('U'))
NameError: name 'np' is not defined
[2022-10-28 08:17:18,610: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:17:18,612: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:17:18,612: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:17:18,612: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:17:18,784: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 08:17:18,793: ERROR: stage_02_featurization]: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 56, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 43, in main
    train_words = np.array(df_train.text.str.lower().values.astype('U'))
AttributeError: 'NoneType' object has no attribute 'text'
[2022-10-28 08:18:19,518: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 08:18:19,519: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 08:18:19,519: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 08:18:19,519: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 08:18:19,685: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 08:18:20,843: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:07:42,805: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:07:42,812: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:07:42,818: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:07:42,819: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:07:43,813: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:07:54,891: INFO: featurize]: The output matrix artifacts\prepared\train.pkl with size (19921, 3002) and data tyoe: float64
[2022-10-28 19:07:55,157: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:10:48,411: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:10:48,413: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:10:48,414: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:10:48,414: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:10:48,605: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:10:59,201: INFO: featurize]: The output matrix artifacts\prepared\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:10:59,453: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:16:49,698: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:16:49,698: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:16:49,698: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:16:49,698: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:16:49,698: ERROR: stage_02_featurization]: [Errno 2] No such file or directory: 'artifacts\\features\\train.tsv'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 66, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 41, in main
    df_train = get_df(train_data_path)
  File "d:\python examples\0.2 dvc\nlp_dvc_demo\src\utils\common.py", line 33, in get_df
    names=['id', 'label', 'text'])
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\parsers\base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\pandas\io\common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts\\features\\train.tsv'
[2022-10-28 19:17:45,752: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:17:45,753: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:17:45,753: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:17:45,753: INFO: common]: created directory at: artifacts\features
[2022-10-28 19:17:45,924: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:17:56,967: INFO: featurize]: The output matrix artifacts\prepared\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:17:57,145: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:19:17,406: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:19:17,406: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:19:17,406: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:19:17,406: INFO: common]: created directory at: artifacts\features
[2022-10-28 19:19:17,580: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:19:26,907: INFO: featurize]: The output matrix artifacts\prepared\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:19:27,051: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:23:19,668: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:23:19,669: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:23:19,669: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:23:19,669: INFO: common]: created directory at: artifacts\features
[2022-10-28 19:23:19,840: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:23:30,773: INFO: featurize]: The output matrix artifacts\features\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:23:30,958: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:33:28,732: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:33:28,733: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:33:28,734: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:33:28,734: INFO: common]: created directory at: artifacts\features
[2022-10-28 19:33:28,924: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:33:39,594: INFO: featurize]: The output matrix artifacts\features\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:33:39,991: INFO: common]: The input data frame is created at artifacts\prepared\test.tsv and size of (5079, 3)
[2022-10-28 19:33:41,066: INFO: featurize]: The output matrix artifacts\features\test.pkl with size (5079, 3002) and data type: float64
[2022-10-28 19:33:41,250: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 19:44:08,600: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 19:44:08,601: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:44:08,602: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:44:08,602: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:44:10,843: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 19:45:17,008: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 19:45:17,010: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:45:17,010: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:45:17,010: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:45:18,922: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 19:45:44,391: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 19:45:44,392: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:45:44,392: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:45:44,392: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 19:45:46,307: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 19:45:49,027: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 19:45:49,029: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 19:45:49,030: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 19:45:49,030: INFO: common]: created directory at: artifacts\features
[2022-10-28 19:45:50,124: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 19:45:59,787: INFO: featurize]: The output matrix artifacts\features\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 19:46:00,091: INFO: common]: The input data frame is created at artifacts\prepared\test.tsv and size of (5079, 3)
[2022-10-28 19:46:01,083: INFO: featurize]: The output matrix artifacts\features\test.pkl with size (5079, 3002) and data type: float64
[2022-10-28 19:46:01,256: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 20:06:51,376: INFO: stage_03_train]: ********* stage three started *********
[2022-10-28 20:06:51,376: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:06:51,380: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 20:06:51,380: INFO: common]: created directory at: artifacts\model
[2022-10-28 20:06:51,402: INFO: stage_03_train]: input matrix size: (19921, 3002)
[2022-10-28 20:06:51,402: INFO: stage_03_train]: X matrix size: (19921, 3000)
[2022-10-28 20:06:51,402: INFO: stage_03_train]: Y matrix size: (19921,)
[2022-10-28 20:06:55,065: INFO: stage_03_train]: ********* stage three completed!*********

[2022-10-28 20:16:25,569: INFO: stage_01_prepare]: ********* stage One started *********
[2022-10-28 20:16:25,571: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:16:25,572: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 20:16:25,572: INFO: common]: created directory at: artifacts\prepared
[2022-10-28 20:16:27,328: INFO: stage_01_prepare]: ********* stage One completed!*********

[2022-10-28 20:16:29,575: INFO: stage_02_featurization]: ********* stage two started *********
[2022-10-28 20:16:29,576: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:16:29,577: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 20:16:29,577: INFO: common]: created directory at: artifacts\features
[2022-10-28 20:16:30,667: INFO: common]: The input data frame is created at artifacts\prepared\train.tsv and size of (19921, 3)
[2022-10-28 20:16:40,406: INFO: featurize]: The output matrix artifacts\features\train.pkl with size (19921, 3002) and data type: float64
[2022-10-28 20:16:40,702: INFO: common]: The input data frame is created at artifacts\prepared\test.tsv and size of (5079, 3)
[2022-10-28 20:16:41,658: INFO: featurize]: The output matrix artifacts\features\test.pkl with size (5079, 3002) and data type: float64
[2022-10-28 20:16:41,846: INFO: stage_02_featurization]: ********* stage two completed!*********

[2022-10-28 20:16:43,359: INFO: stage_03_train]: ********* stage three started *********
[2022-10-28 20:16:43,359: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:16:43,361: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 20:16:43,361: INFO: common]: created directory at: artifacts\model
[2022-10-28 20:16:43,376: INFO: stage_03_train]: input matrix size: (19921, 3002)
[2022-10-28 20:16:43,376: INFO: stage_03_train]: X matrix size: (19921, 3000)
[2022-10-28 20:16:43,376: INFO: stage_03_train]: Y matrix size: (19921,)
[2022-10-28 20:16:46,179: INFO: stage_03_train]: ********* stage three completed!*********

[2022-10-28 20:17:18,650: INFO: stage_03_train]: ********* stage three started *********
[2022-10-28 20:17:18,651: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:17:18,652: INFO: common]: yaml file: params.yaml loaded successfully
[2022-10-28 20:17:18,652: INFO: common]: created directory at: artifacts\model
[2022-10-28 20:17:18,663: INFO: stage_03_train]: input matrix size: (19921, 3002)
[2022-10-28 20:17:18,664: INFO: stage_03_train]: X matrix size: (19921, 3000)
[2022-10-28 20:17:18,664: INFO: stage_03_train]: Y matrix size: (19921,)
[2022-10-28 20:17:21,439: INFO: stage_03_train]: ********* stage three completed!*********

[2022-10-28 20:43:51,626: INFO: stage_04_evaluate]: ********* stage four started *********
[2022-10-28 20:43:51,627: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 20:43:52,336: INFO: stage_04_evaluate]: ********* stage four completed!*********

[2022-10-28 22:34:25,130: INFO: stage_04_evaluate]: ********* stage four started *********
[2022-10-28 22:34:25,133: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 22:34:25,376: ERROR: stage_04_evaluate]: Found input variables with inconsistent numbers of samples: [5079, 1]
Traceback (most recent call last):
  File "src/stage_04_evaluate.py", line 76, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_04_evaluate.py", line 44, in main
    avg_prec = metrics.average_precision_score(labels, predictions)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 233, in average_precision_score
    average_precision, y_true, y_score, average, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 206, in _binary_uninterpolated_average_precision
    y_true, y_score, pos_label=pos_label, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 859, in precision_recall_curve
    y_true, probas_pred, pos_label=pos_label, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 733, in _binary_clf_curve
    check_consistent_length(y_true, y_score, sample_weight)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\utils\validation.py", line 334, in check_consistent_length
    % [int(l) for l in lengths]
ValueError: Found input variables with inconsistent numbers of samples: [5079, 1]
[2022-10-28 22:35:47,586: INFO: stage_04_evaluate]: ********* stage four started *********
[2022-10-28 22:35:47,586: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-10-28 22:35:47,842: ERROR: stage_04_evaluate]: Found input variables with inconsistent numbers of samples: [5079, 1]
Traceback (most recent call last):
  File "src/stage_04_evaluate.py", line 76, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_04_evaluate.py", line 44, in main
    avg_prec = metrics.average_precision_score(labels, predictions)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 233, in average_precision_score
    average_precision, y_true, y_score, average, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 206, in _binary_uninterpolated_average_precision
    y_true, y_score, pos_label=pos_label, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 859, in precision_recall_curve
    y_true, probas_pred, pos_label=pos_label, sample_weight=sample_weight
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\metrics\_ranking.py", line 733, in _binary_clf_curve
    check_consistent_length(y_true, y_score, sample_weight)
  File "C:\Users\luqman\anaconda3\envs\nlp_dvc\lib\site-packages\sklearn\utils\validation.py", line 334, in check_consistent_length
    % [int(l) for l in lengths]
ValueError: Found input variables with inconsistent numbers of samples: [5079, 1]
